cross-glibc-devel-inject.spec

```diff
@@ -149,6 +149,7 @@ shellquote "arch %{_target_cpu} targets i486:%{_target_cpu} x86_64:%{_target_cpu
 shellquote "%{name}" >> /tmp/baselibs_new.conf
 shellquote "  targettype x86 block!" >> /tmp/baselibs_new.conf
 shellquote "  targettype 32bit block!" >> /tmp/baselibs_new.conf
+shellquote "  targettype 64bit block!" >> /tmp/baselibs_new.conf
```

Comment:
Why are we suddenly blocking 64bit?

Reply:
Without this on aarch64 OBS build of cross inject package build rpms for aarch64_ilp32 appear (without the dontuse in their name) which causes issues with aarch64 builds because two packages will provide same content, this issue only appears for libxcrypt because project config on OBS luckily prevents it fro glibc.